sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala

```diff
@@ -230,7 +232,8 @@ private[hive] trait HiveStrategies {
           case o: LogicalPlan =>
             ExecutedCommand(RunnableDescribeCommand(planLater(o), describe.output)) :: Nil
         }
-
+      case drop: LogicalDropTable =>
+        ExecutedCommand(HiveDropTableCommand(drop.tableName, drop.isExists)) :: Nil
```

Comment:
Once we add `isTemporary` to `o.a.s.s.hive.execution.DropTable`, you don't need this `case` branch.